docs/copied-from-beats/security/users.asciidoc

```diff
@@ -4,198 +4,311 @@
+set to `false`):
++
 [options="header"]
 |====
-|Task | Required privileges and roles
+|Privileges | Why needed?
 
 ifndef::apm-server[]
-.3+|Send data to a secured cluster without index lifecycle management
-|`monitor` on cluster
-ifeval::["{beatname_lc}"=="filebeat"]
-(and `manage_pipeline` if {beatname_uc} modules are used)
-endif::[]
-|`create_index` and `index` on +{beat_default_index_prefix}-*+ indices
-|also requires privileges to <<privileges-to-setup-beats,set up index templates>>
-unless you've disabled automatic template loading
-
-.2+|Send data to a secured cluster that supports index lifecycle management
-|`manage_index_templates`, `manage_ilm`
-footnote:[Use `read_ilm` instead of `manage_ilm` if you pre-loaded the lifecycle policy]
-, and `monitor` on cluster
+|`monitor`
+|Send monitoring info
+endif::apm-server[]
+
+ifndef::no_ilm[]
+|`read_ilm`
+|Read the ILM policy when connecting to clusters that support ILM
+endif::no_ilm[]
+
 ifeval::["{beatname_lc}"=="filebeat"]
-(and `manage_pipeline` if {beatname_uc} modules are used)
+|`manage_pipeline`
+|Load ingest pipelines used by modules
 endif::[]
-| `index` and `manage` on +{beat_default_index_prefix}-*+ indices
-endif::apm-server[]
 
-ifdef::apm-server[]
-.3+|Send data to a secured cluster without index lifecycle management
-|`monitor` on cluster
-|`create_index` and `write` on +{beat_default_index_prefix}-*+ indices
-|also requires privileges to <<privileges-to-setup-beats,set up index templates>>
-unless you've disabled automatic template loading: `setup.template.enabled=false`
-
-.3+|Send data to a secured cluster that supports index lifecycle management
-|`manage_ilm` and `monitor` on cluster
-| `index` and `manage` on +{beat_default_index_prefix}-*+ indices
-|also requires privileges to <<privileges-to-setup-beats,set up index templates>>
-unless you've disabled automatic template loading: `setup.template.enabled=false`
-endif::apm-server[]
+ifndef::no_ilm[]
+|`view_index_metadata` on +{beat_default_index_prefix}-*+ indices
+|Check for alias when connecting to clusters that support ILM
+endif::no_ilm[]
 
-ifdef::has_central_config[]
-.2+|Read configurations from Beats central management
-| `monitor` on cluster
-|`beats_admin` role
-endif::has_central_config[]
+|`index` on +{beat_default_index_prefix}-*+ indices
+|Index events into {es}
+
+|`create_index` on +{beat_default_index_prefix}-*+ indices
+|Create daily indices when connecting to clusters that do not support ILM
 |====
+ifndef::apm-server[]
++
+Omit any privileges that aren't relevant in your environment.
+endif::apm-server[]
 
-// REVIEWERS: Do users need `index` and `manage` on `shrink-beatname-*`, too?
-// Are there other privileges that might be required as indices move through the
-// lifecycle stages?
+. Assign the *writer role* to users who will index events into {es}. 
 
 [[kibana-user-privileges]]
-==== Privileges needed by {kib} users
+==== Grant privileges and roles needed to read {beatname_uc} data
```

Comment:
I was expecting to see agent configuration management to be covered here as well, or at least linked to since this is a shared doc - thoughts?

Reply:
Yup, good catch. Luckily this is already in our own if block, so an easy add:
<img width="744" alt="Screen Shot 2019-08-15 at 9 28 38 AM" src="https://user-images.githubusercontent.com/5618806/63109964-34ba9a00-bf3f-11e9-868e-5273a718074d.png">